app/src/main/res/values-fr/strings.xml

```diff
@@ -169,6 +169,8 @@
     <string name="suffix_osc_opacity">%</string>
 
     <string name="category_ui_settings">Paramètres de l\'interface utilisateur</string>
+    <string name="title_checkbox_enable_pip">Activez le mode d’observation pip</string>
+    <string name="summary_checkbox_enable_pip">Permet de visualisation du flux (mais non contrôlé) tout en multitâche</string>
```

Comment:
The build is failing because these are defined twice in this file.

Reply:
Ah damn sorry I had not seen excuse me for this mistake